src/assets/translations/kg.json

```diff
@@ -29288,7 +29288,7 @@
                   "verse": "12"
                 },
                 {
-                  "text": "Ti engem így hívtok: Mester, és Uram . És jól mondjátok, mert az vagyok.",
+                  "text": "Ti engem így hívtok: Mester, és Uram. És jól mondjátok, mert az vagyok.",
```

Comment:
Fixing the response of szentiras.hu API:
https://szentiras.hu/api/idezet/Jn13,13/KG

I checked the text version, there is a text reference symbol before '.', so the text version of KB is OK.
This text fix should be documented if the PR is accepted.